### DISCUSSION TITLE
PGlite + ts-node issues

### DISCUSSION THREAD

**Discussion by sombriks** [electric-sql/pglite#117] [Q&A]
Hello all, 

i am facing issues trying to use PGlite into a sample ts-node project.

i had to disable "type":"module"  and now looks like i have to enable it again, but i am unsure about  the proper setup on this.

this is the sample project:
https://github.com/sombriks/node-backend-validation/tree/main/04-address-book-ts-node

any guidance is welcome on this setup.

**Comment by sombriks** [comment]
small update, so far the only way to make it work is to bail out ts-node and [use tsx instead](https://github.com/sombriks/node-backend-validation/blob/4311c859f02b6a672052ca5bd9e994eb5d8dfa10/04-address-book-ts-node/package.json#L13)

**Reply by samwillis** [reply]
Hey @sombriks 

Thanks for the report on this. With most WASM projects there are often problems around bundlers / ts compilers. I'm keen to tackle these problems next as soon as we get v0.2 final out (we have just released an alpha).

I'm going to spend some time look at this and CJS bundling then.

If you have any insight from your experiments I would love to hear.

**Reply by sombriks** [reply]
That's great news!

As soon as i get a spare time i'll write some exploration code on this topic!
